### ISSUE TITLE
Exported fetch declaration is slightly non-standard

### ISSUE THREAD

**Issue by maxnachlinger** [apollographql/apollo-server#4741]
Found in: [`apollo-server-env@master`](https://github.com/apollographql/apollo-server/blob/a26c08dc43fb7896407d90be17b7b8befdbc73f0/packages/apollo-server-env/src/global.d.ts#L2)

Expected: The exported `fetch` function declaration adheres to the `fetch` standard.

Actual: It differs from the `fetch` standard via marking the initial argument as optional.
```typescript
declare function fetch(
  input?: RequestInfo, // <--- this is what I'm on about
  init?: RequestInit,
): Promise<Response>;
```

Background:

IMHO the exported `fetch` function shouldn't allow the first argument to be omitted. [The MDN fetch page](https://developer.mozilla.org/en-US/docs/Web/API/WindowOrWorkerGlobalScope/fetch) shows this arg as required as does [the living fetch standard](https://fetch.spec.whatwg.org/#requestinfo) - `fetch` args are really `Request` ctor args :) 

Why am I bothering you about this?
I've created my own fun `fetch` function and want to pass it along to a `RemoteGraphQLDataSource` , since the first arg to my `fetch` function isn't optional, `tsc` is complaining.

**Comment by maxnachlinger** [comment]
Hey thanks for the fix!

**Comment by trevor-scheer** [comment]
Thanks for the report! The fix is released via commit c212627. You'll see the latest versions for whichever packages interest you listed in the commit message.
